core/src/epicli/data/common/ansible/playbooks/roles/vault/files/vault/unseal-vault.sh

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env bash
+
+function log_and_print {
+    local string_to_log="$1";
+    echo "$(date +"%Y-%m-%d-%H:%M:%S"): $string_to_log" | tee -a /opt/vault/logs/unseal_vault.log;
+}
+
+function exit_with_error {
+    local string_to_log="$1";
+    log_and_print "ERROR: $string_to_log";
+    exit 1;
+}
+
+function check_vault_error {
+    local exit_code="$1";
+    local success_message="$2";
+    local failure_message="$3";
+    if [ "$exit_code" == "0" ] ; then
+        log_and_print "$success_message";
+    else
+        exit_with_error "$failure_message Exit status: $exit_code";
+    fi
+}
+
+function check_if_vault_is_initialized {
+    vault status -format json | grep -q '"initialized": true';
+    local command_result=( "${PIPESTATUS[@]}" );
+    if [ "${command_result[0]}" = "1" ] ; then
+        exit_with_error "There was an error during checking status of Vault.";
+    fi
+    if [ "${command_result[1]}" = "0" ] ; then
+        log_and_print "Vault is initialized.";
+    elif [ "${command_result[1]}" = "1" ] ; then
+        log_and_print "Vault is not initialized, nothing to unseal.";
+        exit 0;
+    fi
+}
+
+function unseal_vault {
+    local init_file_path="$1";
+    log_and_print "Checking if vault is already unsealed...";
+    vault status -format json | grep -q '"sealed": false';
+    local command_result=( "${PIPESTATUS[@]}" );
+    if [ "${command_result[0]}" = "1" ] ; then
+        exit_with_error "There was an error during checking status of Vault.";
+    fi
+    if [ "${command_result[1]}" = "0" ] ; then
+        log_and_print "Vault is already unsealed. Nothing to do.";
+    elif [ "${command_result[1]}" = "1" ] ; then
+        log_and_print "Unsealing Vault...";
+        grep --max-count=3 Unseal "$init_file_path" | awk '{print $4}' | while read -r line ; do
```

Comment:
Magic number. `init` should be called with value and it should be passed here.

Reply:
Will be done after MVP.